From 3a2650a878b814735430b1b7a25fd190dfacf22e Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Fri, 30 Aug 2019 17:09:23 +0200
Subject: [PATCH] ACPI / property: Fix acpi_graph_get_remote_endpoint() name in kerneldoc
Git-commit: 3a2650a878b814735430b1b7a25fd190dfacf22e
Patch-mainline: v5.4-rc1
References: bsc#1051510

The function is called acpi_graph_get_remote_endpoint(), not
acpi_graph_get_remote_enpoint().

Fixes: 79389a83bc3888a9 ("ACPI / property: Add support for remote endpoints")
Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/acpi/property.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/acpi/property.c
+++ b/drivers/acpi/property.c
@@ -1066,7 +1066,7 @@ static struct fwnode_handle *acpi_graph_
 
 
 /**
- * acpi_graph_get_remote_enpoint - Parses and returns remote end of an endpoint
+ * acpi_graph_get_remote_endpoint - Parses and returns remote end of an endpoint
  * @fwnode: Endpoint firmware node pointing to a remote device
  * @parent: Firmware node of remote port parent is filled here if not %NULL
  * @port: Firmware node of remote port is filled here if not %NULL
